### PR TITLE
Switch asyncComponent to React Lazy/Suspense (Issue #1577)

### PR DESCRIPTION
**Pull request by ankitmhn**
### PR Checklist

- [x] Brief description of changes
Switched out AsyncComponent in favour of React Lazy components in 4 extensions.
- [x] Links to any relevant issues
https://github.com/OHIF/Viewers/issues/1577
- [x] Required status checks are passing
yarn test is passing
- [x] User cases if changes impact the user's experience
N/A
- [x] `@mention` a maintainer to request a review
@swederik 

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by swederik** [comment]
It looks good to me and the deploy preview seems to work fine. I'm not sure what is causing the PR_CHECKS to fail.

**Comment by swederik** [comment]
I'm getting the following error when I run it locally and drag/drop a PDF file into /local:

```
react-hot-loader.development.js?cb82:2406 Uncaught Error: Element type is invalid. Received a promise that resolves to: [object Object]. Lazy element type must resolve to a class or function.
    at mountLazyComponent (react-dom.development.js?730b:17428)
    at beginWork (react-dom.development.js?730b:18635)
    at HTMLUnknownElement.callCallback (react-dom.development.js?730b:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?730b:237)
    at invokeGuardedCallback (react-dom.development.js?730b:292)
    at beginWork$1 (react-dom.development.js?730b:23234)
    at performUnitOfWork (react-dom.development.js?730b:22185)
    at workLoopSync (react-dom.development.js?730b:22161)
    at performSyncWorkOnRoot (react-dom.development.js?730b:21787)
    at eval (react-dom.development.js?730b:11111)
``` 

I've attached the sample DCM inside a zip file.
[sample.dcm.zip](https://github.com/OHIF/Viewers/files/4408639/sample.dcm.zip)

File: extensions/dicom-pdf/src/index.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+import React from 'react';
 import OHIFDicomPDFSopClassHandler from './OHIFDicomPDFSopClassHandler.js';
 
-const ConnectedOHIFDicomPDFViewer = asyncComponent(() =>
-  import(
-    /* webpackChunkName: "ConnectedOHIFDicomPDFViewer" */ './ConnectedOHIFDicomPDFViewer'
-  )
-);
+const Component = React.lazy(() => {
+  return import('./OHIFDicomPDFSopClassHandler');
```
Comment (swederik) [inline extensions/dicom-pdf/src/index.js]: This doesn't seem right. That's not a component, it's just a function. You would need to put ConnectedOHIFDicomPDFViewer in here instead and rename the function on line 8, I suppose.

Comment (ankitmhn) [reply extensions/dicom-pdf/src/index.js]: Okay. I have updated the file with the correct component.

Comment (swederik) [reply extensions/dicom-pdf/src/index.js]: Thanks, it's appears to be fixed now.

**Comment by swederik** [comment]
Thanks!

**Pull request merged**